x-pack/plugins/apm/dev_docs/testing.md

```diff
@@ -129,6 +129,18 @@ APM tests are located in `x-pack/test/functional/apps/apm`.
 For debugging access Elasticsearch on http://localhost:9220` (elastic/changeme)
 diff --git a/x-pack/plugins/apm/scripts/test/README.md b/x-pack/plugins/apm/scripts/test/README.md
 
+## Serverless API tests
+
+**Start server**
+```
+node scripts/functional_tests_server.js --config x-pack/test_serverless/api_integration/test_suites/observability/config.ts
+```
+
+**Run tests**
+```
+node scripts/functional_test_runner --config=x-pack/test_serverless/api_integration/test_suites/observability/config.ts
+```
+
```

Comment:
```suggestion
**Single Script to run tests
```
node scripts/functional_tests.js --config x-pack/test_serverless/api_integration/test_suites/observability/config.ts
```

**Running tests against MKI**
```
TEST_CLOUD=1 TEST_ES_URL=https://elastic:JX5WO0745qKzC6FASQ3288bp@e3ff6fd04a9d4791a065aee3de66c063.es.qa-awseuw1-cp-app-1.qa.cld.elstc.co:443 TEST_KIBANA_URL=https://elastic:<PASSWORD>@<KIBANA_URL> node  scripts/functional_test_runner --config x-pack/test_serverless/api_integration/test_suites/observability/config.ts
```
```

Reply:
What are the steps to get  `PASSWORD` and `ES_URL `?

Reply:
A new project needs to be created at `https://console.qa.cld.elstc.co/home` and that would provide the `ES_URL` and `KIBANA_URL`.

Password needs to be reset using the Reset API, 

- Create a project in Cloud Console and after creation, copy the projectId from the URL in your browser (P.S. Project ID is not same as the id present in Kibana/ES URL)

- Generate a API Key in Cloud Console
- Call this endpoint - [https://global.qa.cld.elstc.co/api/v1/serverless/projects/observability/{YOUR_PROJECT_ID}/_reset-credentials](https://global.qa.cld.elstc.co/api/v1/serverless/projects/observability/%7BYOUR_PROJECT_ID%7D/_reset-credentials).
- This Endpoint requires Authorisation header with API Key - https://www.elastic.co/guide/en/cloud/current/ec-api-authentication.html

Reply:
I don't think mentioning the MKI without the additional context makes sense. OTOH I'm not sure if we should add these details to the public docs. Yet another reason to move to the internal Dev Docs.